src/cljs/rems/application.cljs

```diff
@@ -539,13 +539,14 @@
 
 (defn- actions-form [application]
   (let [app-id (:application/id application)
+        see-everything? (contains? (:application/permissions application) :see-everything)
         actions (action-buttons application)
         reload (partial reload! app-id)
         forms [[:div#actions-forms.mt-3
                 [request-comment-form app-id reload]
                 [request-decision-form app-id reload]
                 [comment-form app-id reload]
-                [close-form app-id reload]
```

Comment:
I find it kinda weird that you use the the `:see-everything` permission as a proxy for showing the comment field. I think it would be cleaner to either a) look at whether the user has the applicant role or b) have a separate close-without-comment permission

Reply:
It's the same permission which is used to determine whether the user is allowed to see the comment: https://github.com/CSCfi/rems/blob/bc6436368eba588b1b40b7a73687a9c007fa8be7/src/clj/rems/application/model.clj#L573-L577 Thus I think it logical that the same permission is used - otherwise the user could write a comment which he is not allowed to see.

Reply:
Ok I'm fine with it if you add a comment. Please also consider renaming the parameter to `close-view` and `close-form` to `show-comment-box?` or something like that.